WooCommerce/woocommerce_rms_normal/class-gateway.php

```diff
@@ -25,6 +25,7 @@ public function __construct() {
     $this->verify_key = $this->settings['verify_key'];
     $this->secret_key = $this->settings['secret_key'];
     $this->account_type = $this->settings['account_type'];
+    $this->recurring = $this->settings['recurring'];
     
     // Define hostname based on account_type
     $this->url = ($this->get_option('account_type')=='1') ? "https://pay.merchant.razer.com/" : "https://sandbox.merchant.razer.com/" ;
```

Comment:
change pay.merchant.razer.com to pay.fiuu.com

Reply:
![image](https://github.com/user-attachments/assets/654e74ce-25e1-4807-a8e2-67ca6f3d8365)